webpack/redux/reducers/RedHatRepositories/index.js

```diff
@@ -1,8 +1,15 @@
 import { combineReducers } from 'redux';
 import enabled from './enabled';
 import sets from './sets';
+import repositorySetRepositories from './repositorySetRepositories';
+
+const orgId = () => ({
+  id: document.getElementById('organization-id').dataset.id,
+});
```

Comment:
Wonder if we want to do this in a common reducer since we will likely need it on other pages.  I am also open to doing this later as we need it.

Reply:
Yeah for sure - didn't realize it was nested here.